### PR TITLE
Improve mc-cp error message

### LINKED ISSUE #2195: Ugly error message when target does not exist
## Expected behaviour
See below, an error message along the lines of "Target 'mygcs/from-holland/xl-v1/go' does not exist" would be nicer.

## Actual behaviour
```
$ mc cp mygcs/from-holland/xl-v1/go .
mc: <ERROR> Unable to stat 'mygcs/from-holland/xl-v1/go'. Object does not exist
```

## System information
Running minio in gateway mode to GCS configured back-end

### PR DESCRIPTION
**Pull request by krisis**
Fixes #2195

### CODE REVIEW

**Comment by krisis** [comment]
Sample output:
```
➜  mc git:(issue/2195) mc cp mygcs/testkp-1/xl-v1/go .
mc: <ERROR> Source mygcs/testkp-1/xl-v1/go doesn't exist.
```

File: cmd/cp-url-syntax.go [hunk 1]
```diff
@@ -42,7 +43,7 @@ func checkCopySyntax(ctx *cli.Context) {
 	for _, srcURL := range srcURLs {
 		_, _, err := url2Stat(srcURL)
 		if err != nil {
-			fatalIf(err.Trace(srcURL), fmt.Sprintf("Unable to stat '%s'.", srcURL))
+			console.Fatalf("Source %s doesn't exist.\n", srcURL)
```
Comment (vadmeste) [inline cmd/cp-url-syntax.go]: Couldn't this be sometimes a network problem ?

We probably can find a better phrase than 'Unable to stat' of this latter is considered not user friendly

Comment (fwessels) [inline cmd/cp-url-syntax.go]: Suggestion: "Unable to determine validity of source %s"

Comment (krisis) [inline cmd/cp-url-syntax.go]: @vadmeste does the above phrase look better? If so, I can update the PR with @fwessels' suggestion.

File: cmd/cp-url-syntax.go [hunk 2]
```diff
@@ -42,7 +43,7 @@ func checkCopySyntax(ctx *cli.Context) {
 	for _, srcURL := range srcURLs {
 		_, _, err := url2Stat(srcURL)
 		if err != nil {
-			fatalIf(err.Trace(srcURL), fmt.Sprintf("Unable to stat '%s'.", srcURL))
+			console.Fatalf("Unable to determine validity of source %s\n", srcURL)
```
Comment (vadmeste) [inline cmd/cp-url-syntax.go]: `Unable to validate source %s`
or
`Unable to check source %s`
or
`Unable to locate source %s`

I think these would be better. I'll approve if you vote for no.

Comment (fwessels) [inline cmd/cp-url-syntax.go]: I like the first option

**Pull request merged**